crates/uv-installer/src/site_packages.rs

```diff
@@ -242,6 +244,16 @@ impl SitePackages {
                     }
                 }
 
+                // Verify that the package is compatible with the current tags.
+                if let Some(wheel_tags) = distribution.read_tags()? {
+                    if !wheel_tags.is_compatible(tags) {
+                        // TODO(charlie): Show the expanded tag hint, that explains _why_ it doesn't match.
```

Comment:
👍 this seems nice to have, might be worth opening an issue for it maybe someone can pick it up